backend/ingest.py

```diff
@@ -15,6 +13,9 @@
 from langchain_core.embeddings import Embeddings
 from langchain_openai import OpenAIEmbeddings
 
+from backend.constants import WEAVIATE_DOCS_INDEX_NAME
+from backend.parser import langchain_docs_extractor
```

Comment:
would be careful about this, vaguely remember that something about the way we deploy makes it so we don't want the `backend.`

Reply:
yea, i think something got messed up in vercel, going to dig deeper